lib/swoc/include/swoc/TextView.h

```diff
@@ -1384,7 +1384,7 @@ TextView::suffix(int n) const noexcept {
 inline TextView
 TextView::suffix_at(char c) const {
   self_type zret;
-  if (auto n = this->rfind(c); n != npos) {
+  if (auto n = this->rfind(c); n != npos && n + 1 < this->size()) {
```

Comment:
This addresses a compiler warning from Centos that zret.assign referenced the byte after data() if n pointed to the last byte in the string.